tests/features/search-form.feature

```diff
@@ -0,0 +1,12 @@
+@api
+Feature: Search form block
```

Comment:
I don't think we need this test here. The oe_search component already makes sure the functionality works.

Reply:
@imanoleguskiza Yes we do. The component tests the functionality but this tests that the homepage has it. This block is the final block that needs to be placed there by the theme on all pages. @sergepavle Potentially we can test that it's available also on other pages no?

Reply:
Then the last 3 steps are not required and neither is the custom context. Agree testing other pages would be nice.

Reply:
@imanoleguskiza Well, it doesn't hurt to have it does it?

Reply:
I have removed 3 steps and added check presents search block on other pages.
Also I have added Kernel tests for testing rendering block with ECL components (like for other elements of a pages like breadcrumbs, language switcher etc.).